aspnetcore/mvc/views/view-compilation.md

```diff
@@ -90,6 +90,25 @@ A *<project_name>.PrecompiledViews.dll* file, containing the compiled Razor file
 
 ::: moniker-end
 
+## Recompile views on file change behavior
+
+The <xref:Microsoft.AspNetCore.Mvc.Razor.RazorViewEngineOptions> `AllowRecompilingViewsOnFileChange` gets or sets a value that determines if Razor files (Razor Views and Razor Pages) are recompiled and updated if files change on disk.
+
+When set to `true`, MVC uses [IFileProvider.Watch](xref:<xref:Microsoft.Extensions.FileProviders.IFileProvider.Watch*>) to watch for changes to Razor files in configured <xref:Microsoft.Extensions.FileProviders.IFileProvider> instances.
```

Comment:
@tdykstra 	@scottaddie 	should this be

When set to `true`, the ASP.NET Core runtime uses ...

Reply:
"MVC uses" is probably OK if it's really the MVC framework doing the watching -- if not, I would prefer your proposal.

Reply:
I suggest the following:

When set to `true`, ASP.NET Core uses...

Reply:
I have another idear ... check the next commit.